Views/Doctors/Delete.cshtml

```diff
@@ -47,6 +47,36 @@
         <dd class = "col-sm-10">
             @Html.DisplayFor(model => model.Speciality)
         </dd>
+        <dt class = "col-sm-2">
```

Comment:
Why this page? Is is related to LOGIN?

Reply:
no .. It is created as a part of scaffold for doctor model